src/raftstore/store/worker/apply.rs

```diff
@@ -2017,14 +2192,18 @@ impl Runner {
             sync_log: sync_log,
             use_delete_range: use_delete_range,
             tag: format!("[store {}]", store.store_id()),
+            sub_apply_worker: sub_apply_worker,
         }
     }
 
     fn handle_applies(&mut self, applys: Vec<Apply>) {
```

Comment:
What if the delegate is paused here?

Reply:
Actually it will never happen, but it is better do some check here.